File: Aplikacija/Front deo/erestoranFront/src/app/feature/restaurants/models/Rezervacija.ts
import { Korisnik } from "./Korisnik";
import { Sto } from "./Sto";

export interface Rezervacija
{
    id: number;
    korisnik: Korisnik;
    sto: Sto;
    brojLjudi: number;
    datumRezervacije: Date;
}

File: Aplikacija/Back deo/ERestoran/Models/Rezervacija.cs
namespace Models
{
    public class Rezervacija
    {
        [Key]
        public int ID { get; set; }
        public Korisnik? Korisnik { get; set; }
        [JsonIgnore]
        public Sto? Sto { get; set; }
        public int BrojLjudi { get; set; }
        //[DisplayFormat(DataFormatString = "{0:dd-MM-yyyy}", ApplyFormatInEditMode = true)]
        public DateTime DatumRezervacije { get; set; }
    }
}